src/components/SignUp/SignUp.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable linebreak-style */
+import React, { useRef } from 'react';
+import { useDispatch } from 'react-redux';
+import { Link, useNavigate } from 'react-router-dom';
+import { postSignUp } from '../../redux/registration/registration';
+import './Signup.css';
+
+const SignUp = () => {
+  const navigate = useNavigate();
+  const formRef = useRef();
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const formData = new FormData(formRef.current);
+    const data = Object.fromEntries(formData);
+    const user = {
+      name: data.name,
+      phone: data.phone,
+      email: data.email,
+      password: data.password,
+      password_confirmation: data.password_confirmation,
+    };
+    /* eslint-disable no-console */
+    console.log(user);
```

Comment:
- [ ] Kindly remove this old piece of debugging console.log statement, in order to improve the repo's cleanliness. 👍